.yamato/_formatting.yml

```diff
@@ -18,3 +18,5 @@ formatting:
       -  hg clone -u stable http://hg-mirror-slo.hq.unity3d.com/unity-extra/unity-meta ~/unity-meta
       -  perl ~/unity-meta/Tools/Format/format.pl --hgroot $(pwd) --dry-run .
     timeout: 1
+    triggers:
+        expression: pull_request.target eq "master" AND NOT pull_request.draft
```

Comment:
is there a particular reason we wouldnt want this to run on a draft?

Reply:
It was to save bokken resources.
Do you think it would be ok to run it at each commit even draft?

Reply:
yeah i think so, especially if we turn on auto-abv again, it might be useful to sort formatting stuff out before triggering abv

Reply:
ok, let me remove the draft part then. thanks.